src/database/sql/audio.py

```diff
@@ -368,7 +398,8 @@ def get_aggressive(self, audio_id: str) -> str:
         query = """
             SELECT aggressive, aggressive_confidence, last_updated
             FROM Audio
-            WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+            WHERE audio_release=:rel AND audio_side=:side
+            AND audio_track=:track
```

Comment:
If you want this AND `audio_track=:track` to be on a new line, please indent it.